src/service/report-pdf-service.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ *  SudoSOS back-end API service.
+ *  Copyright (C) 2024  Study association GEWIS
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published
+ *  by the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import {
+  Client, FileResponse,
+  FileSettings,
+  FineReportParameters,
+  FineRouteParams,
+  IFileSettings,
+  Language,
+  Product,
+  ProductPricing,
+  ReturnFileType,
+  TotalPricing,
+  VAT,
+} from 'pdf-generator-client';
+import { FineReport } from '../controller/response/debtor-response';
+import { PDF_GEN_URL, PDF_VAT_HIGH, UNUSED_NUMBER, UNUSED_PARAM } from '../helpers/pdf';
+
+const HANDED_OUT_FINES = 'Handed out';
+const WAIVED_FINES = 'Waived';
+
+export default class ReportPdfService {
```

Comment:
Is this class only for fine reports? Because then the name should be different

Reply:
My intention is to have it for all reports in pdf